config.yaml

```
@@ -34,13 +34,13 @@ life_cycle: 'pre-alpha' # FIXME
 license: 'CC-BY 4.0'
 
 # Link to the source repository for this lesson
-source: 'https://github.com/carpentries/workbench-template-rmd' # FIXME
+source: 'https://erinbecker.github.io/lesson-example-R/' # FIXME
```

```suggestion
source: 'https://github.com/erinbecker/lesson-example-R/'
```

`source` should contain the URL of your lesson repository rather than the lesson itself